Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -0,0 +1,423 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+
+{#
+
+This code manages the many-to-[one|many] association field popup
+
+#}
+
+{% autoescape false %}
+
+{% set associationadmin = sonata_admin.field_description.associationadmin %}
+
+<!-- edit many association -->
+
+<script type="text/javascript">
```

Comment:
So lets. merge and tag, i will start with the PR on doctrine phpcr admin and if nobody else likes to do it, with that knowledge on orm admin.